sources/web/datalab/templates/tree.html

```diff
@@ -86,11 +86,14 @@
                             <span class="fa fa-files-o"></span> Copy
                           </button>
                           <button id="renameButton" type="button" class="toolbar-btn rename-button" title="Rename selected item(s)">
-                            <span class="fa fa-edit"></span> Rename
+                            <span class="fa fa-text-height"></span> Rename
```

Comment:
I don't see how 'fa-text-height' indicates renaming.

How about, instead, keeping the 'fa-edit' icon here, and for the text editor use the 'file-text-o' icon. That way, we are indicating that button opens the file as a text file.